src/makeDOMDriver.ts

```diff
@@ -62,8 +63,13 @@ function makeDOMDriver(container: string | Element, options?: DOMDriverOptions):
     /* tslint:disable:no-empty */
     rootElement$.addListener({next: () => {}, error: () => {}, complete: () => {}});
     /* tslint:enable:no-empty */
-
-    return new DOMSource(rootElement$, runStreamAdapter, [], isolateModule, delegators);
+    const source = new DOMSource(rootElement$, runStreamAdapter, [], isolateModule, delegators);
```

Comment:
Need to update with the latest on diversity branch because now we are using `MainDOMSource`